packages/app/src/components/button/icon/on/index.tsx

```diff
@@ -0,0 +1,38 @@
+import classnames from 'classnames';
+import React, { useCallback } from 'react';
+import { Props as BaseProps } from '~/components';
+import { Props as BaseButtonProps, SIZE } from '~/components/button';
+
+export type Props<T = null> = BaseProps & BaseButtonProps<T>;
+const IconButton = function <T = null>({
+  on,
+  className = '',
+  disabled = false,
+  type = 'button',
+  size = SIZE.BASE,
+  data,
+  onClick,
+  children,
+}: React.PropsWithChildren<Props<T>>): JSX.Element {
+  const handleClick = useCallback(() => {
+    onClick(data as T);
+  }, [data, onClick]);
+
+  return (
+    <button
+      className={classnames(
+        `flex items-center justify-center disabled:pointer-events-none disabled:opacity-60 transition-color hover:bg-thm-on-${on}-slight active:bg-thm-on-${on}-low outline-thm-outline`,
+        className,
+        {
+          'h-11 w-11 text-xl rounded': size === SIZE.BASE,
```

Comment:
I think whether 'rounded' is added or not should be determined by the rounded props, not the size props.
```suggestion
          'rounded-full': rounded,
          'h-11 w-11 text-xl': size === SIZE.BASE,
```

Reply:
fb44a2285402fafb47ec6040c762ef1768ab481d
Fixed!